graph-commons/src/main/scala/io/renku/http/client/GitLabClient.scala

```diff
@@ -56,6 +56,10 @@ trait GitLabClient[F[_]] {
       mapResponse: ResponseMappingF[F, ResultType]
   )(implicit maybeAccessToken: Option[AccessToken]): F[ResultType]
 
+  def postMultipart[ResultType](path: Uri, endpointName: String Refined NonEmpty, payload: Multipart[F])(
```

Comment:
I think at some point in the future, it would be nice to have a "real" GitLab-client, not only a generic http clietn with better logging. Now we need to create the correct request at the call site which means every caller must be aware of all the GitLab request details.

Reply:
I know. I think we were talking about that the other day. I was tempted to maybe do it in the PR but it already grew so big that now I'm happy I didn't do it. I'll create an issue for that.